api/app.py

```diff
@@ -38,9 +41,27 @@ def create_exchange():
 
 
 # user creates a bot
-@app.route("/bots", methods=["POST"])
+@app.route("/bots", methods=["GET"])
```

Comment:
Why we have a `GET` here? We are creating bot, please read HTTP methods description.